recipe/meta.yaml

```
@@ -31,7 +31,7 @@ outputs:
       run:
         - python
         - filelock
-        - huggingface_hub >=0.23.2
+        - huggingface_hub >=0.27.0
```

```suggestion
        - huggingface_hub >=0.23.2
```
This was actually changed to 0.27.0 after 0.32.2 was released.
[huggingface/diffusers@`v0.32.2...main`#diff-85328ed60a](https://github.com/huggingface/diffusers/compare/v0.32.2...main#diff-85328ed60ae8e7412539358c39ea70c93adb33913830705094f0499432070e0e)